### PR TITLE
add eventhub integration test

### PR DESCRIPTION
**Pull request by jacko9et**

### CODE REVIEW

File: sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/microsoft/azure/eventhubs/EventhubIntegrationTests.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    private String connectionString;
+
+    @Value("${spring.cloud.stream.bindings.input.destination}")
+    private String eventHubName;
+
+    @Test
+    void integrationTest(@Autowired Source source) {
+        source.output().send(new GenericMessage<>(MESSAGE));
+
+        EventHubConsumerClient consumerClient = new EventHubClientBuilder()
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/microsoft/azure/eventhubs/EventhubIntegrationTests.java:35]: We can't do integration this way, this is the integration test for EventHub client, not for Spring Cloud Stream Binder.

File: sdk/spring/azure-spring-cloud-test-eventhubs/.gitignore [hunk 2]
```diff
@@ -0,0 +1,32 @@
+target/
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-test-eventhubs/.gitignore:1]: This gitignore is not necessary.

File: sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/microsoft/azure/eventhubs/EventHubIntegrationTests.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.messaging.Source;
+import org.springframework.messaging.support.GenericMessage;
+
+@EnableBinding(Source.class)
+@SpringBootTest
+class EventHubIntegrationTests {
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/microsoft/azure/eventhubs/EventHubIntegrationTests.java:16]: Integration test should ends with `IT`?

**Comment by chenrujun** [comment]
Hi, @lu-cheng .
integration-test pipeline failed, please investigate. 🙏

**Comment by chenrujun** [comment]
/azp run java - spring - tests

File: sdk/spring/azure-spring-cloud-test-eventhubs/pom.xml [hunk 4]
```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>spring-cloud-azure-eventhubs-stream-binder</artifactId>
-      <version>1.2.7</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-eventhubs-stream-binder;external_dependency} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-test-eventhubs/pom.xml]: please also make sure we've removed it from the external_dependency.txt file

**Pull request merged**